authority/provisioner/aws_test.go

```diff
@@ -896,5 +896,5 @@ func TestAWS_HardcodedCertificates(t *testing.T) {
 		assert.True(t, cert.NotAfter.After(time.Now()))
 		certs = append(certs, cert)
 	}
-	assert.Len(t, 14, certs, "expected 14 certificates in aws_certificates.pem")
+	assert.Len(t, 15, certs, "expected 14 certificates in aws_certificates.pem")
```

Comment:
```suggestion
	assert.Len(t, 15, certs, "expected 15 certificates in aws_certificates.pem")
```

Reply:
Good catch, thank you, I will fix this one right away